torchvision/transforms/functional.py

```diff
@@ -8,6 +8,7 @@
 except ImportError:
     accimage = None
 import numpy as np
+from numpy import sin, cos, tan
```

Comment:
nit: Any reason to use numpy's functions instead of those from `math`?

Reply:
No strong reason here - my reason for this import was not much of whether to use math or numpy, but more that I felt this type of code is very error prone, and it was worth it to import the names directly and make the formulae more readable. And then for importing from numpy instead of math, it was just because the numpy functions are a superset of the math ones in terms of behavior (i.e. np.sin works for normal numbers and for np.array while math won't work for np.array), in case the functions are needed for np.arrays in the future.